divan/chat/auth_firebase.py

```diff
@@ -0,0 +1,25 @@
+import pyrebase
+from .config import firebaseConfig
+
+firebase = pyrebase.initialize_app(firebaseConfig)
+auth = firebase.auth()
+
+
+def refresh_token(refresh_token):
+    refresh = auth.refresh(refresh_token)
+    return refresh
```

Comment:
Better call it new_token
```suggestion
    new_token = auth.refresh(refresh_token)
    return new_token
```

Reply:
done